.eslintrc

```diff
@@ -0,0 +1,206 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "ecmaFeatures": {
```

Comment:
be mindful of copy / paste.  

We shouldn't be including 
1.  `"ecmaFeatures.jsx"` doesn't apply since we're not using React
1.  `"env.browser"` shouldn't apply since this code is not meant for the browser